wrapper/src/main/java/software/amazon/jdbc/ConnectionPlugin.java

```diff
@@ -93,7 +94,10 @@ Connection connect(
    * @return a {@link Connection} to the requested host
    * @throws SQLException if there was an error establishing a {@link Connection} to the requested
    *                      host
+   *
+   * @deprecated Use {@link ConnectionService#createAuxiliaryConnection(HostSpec, Properties)} instead.
    */
+  @Deprecated
```

Comment:
Hmm we have plugins (not monitors) that need this pipeline. InitialConnection is one of them.

Reply:
I checked on main and AuroraInitialConnectionStrategyPlugin does not implement forceConnect. In this PR, all of the plugins that implemented forceConnect have been changed to use ConnectionService#createAuxiliaryConnection, which is similar to forceConnect but slightly different (see my comment below for a more detailed explanation)